Disambiguated text.txt

```diff
@@ -0,0 +1,1585 @@
+	"information" n sg
+"<.>"
+	"." sent
+"<Many>"
+	"many" det qnt pl
+;	"many" prn ind mf pl
+"<computers>"
+	"computer" n pl
+"<can>"
+;	"can" n sg
+	"can" vbmod inf
+;	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<store>"
+;	"store" n sg
+	"store" vblex inf
+;	"store" vblex pres
+;	"store" vblex imp
+"<and>"
+	"and" cnjcoo
+"<retrieve>"
+	"retrieve" vblex inf
+;	"retrieve" vblex pres
+;	"retrieve" vblex imp
+"<information>"
+	"information" n sg
+"<using>"
+	"use" vblex pprs
+;	"use" vblex subs
+;	"use" vblex ger
+"<hard>"
+;	"hard" n sg
+	"hard" adj sint
+"<drives>"
+	"drive" n pl
+;	"drive" vblex pres p3 sg
+"<.>"
+	"." sent
+"<Computers>"
+	"computer" n pl
+"<can>"
+;	"can" n sg
+	"can" vbmod inf
+;	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<be>"
+	"be" vbser inf
+"<connected>"
+	"connect" vblex pp
+;	"connect" vblex past
+;	"connected" adj
+"<together>"
+;	"together" adj
+	"together" adv
+"<to>"
+	"to" pr
+"<form>"
+;	"form" n sg
+	"form" vblex inf
+;	"form" vblex pres
+;	"form" vblex imp
+"<networks>"
+	"network" n pl
+;	"network" vblex pres p3 sg
+"<,>"
+	"," cm
+"<allowing>"
+	"allow" vblex pprs
+;	"allow" vblex subs
+;	"allow" vblex ger
+"<connected>"
+;	"connect" vblex pp
+;	"connect" vblex past
+	"connected" adj
+"<computers>"
+	"computer" n pl
+"<to>"
+	"to" pr
+"<communicate>"
+	"communicate" vblex inf
+;	"communicate" vblex pres
+;	"communicate" vblex imp
+"<with>"
+	"with" pr
+"<each>"
+	"each" det ind sg
+;	"each" prn ind mf sg
+"<other>"
+	"other" det ind sp
+"<..>"
+	".." sent
+
+"<The>"
+	"the" det def sp
+"<two>"
+	"two" num pl
+"<principal>"
+	"principal" adj
+;	"principal" n sg
+"<characteristics>"
+	"characteristic" n pl
+"<of>"
+	"of" pr
+"<a>"
+	"a" det ind sg
+"<computer>"
+;	"computer" adj
+	"computer" n sg
+"<are>"
+	"be" vbser pres
+"<:>"
+	":" sent
+"<It>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<responds>"
+	"respond" vblex pres p3 sg
+"<to>"
+	"to" pr
+"<a>"
+	"a" det ind sg
+"<specific>"
+	"specific" adj
+"<instruction>"
+	"instruction" n sg
+"<set>"
+	"set" n sg
+;	"set" vblex inf
+;	"set" vblex pres
+;	"set" vblex past
+;	"set" vblex pp
+;	"set" vblex imp
+"<in>"
+;	"in" adv
+	"in" pr
+"<a>"
+	"a" det ind sg
+"<well>"
+	"well" adv
+;	"well" n sg
+"<->"
+	"-" guio
+"<defined>"
+	"defined" adj
+;	"define" vblex past
+;	"define" vblex pp
+"<manner>"
+	"manner" n sg
+"<and>"
+	"and" cnjcoo
+"<it>"
+	"prpers" prn subj p3 nt sg
+;	"prpers" prn obj p3 nt sg
+"<can>"
+;	"can" n sg
+	"can" vbmod inf
+;	"can" vbmod pres
+;	"can" vblex inf
+;	"can" vblex pres
+;	"can" vblex imp
+"<execute>"
+	"execute" vblex inf
+;	"execute" vblex pres
+;	"execute" vblex imp
+"<a>"
+	"a" det ind sg
+"<prerecorded>"
+	"*prerecorded"
+"<list>"
+	"list" n sg
+;	"list" vblex inf
+;	"list" vblex pres
+;	"list" vblex imp
+"<of>"
+	"of" pr
+"<instructions>"
+	"instruction" n pl
+"<call>"
+;	"call" n sg
+	"call" vblex inf
+;	"call" vblex pres
+;	"call" vblex imp
```

Comment:
Odd, I would have expected the text to have "called" here (`<pp>`)

Reply:
What should I do in this case?
I have verified that it is an error in the text source only(It is from wikipedia)
Should I correct it like this:
"<called>"
      "called"  vblex pp